flash_attn/flash_attn_triton_amd.py

```diff
@@ -1577,7 +1577,72 @@ def fwd(q,
 
     return tri_out, q , k , v, o, softmax_lse, softmax_p, torch.get_rng_state()
 
+def varlen_fwd(
+        q, 
+        k, 
+        v, 
+        o,
+        cu_seqlens_q,
+        cu_seqlens_k,
+        seqused_k,
+        block_table_,
+        alibi_slopes,\
+        max_seqlen_q,
+        max_seqlen_k,
+        dropout_p,
+        softmax_scale,
+        zero_tensors,
+        causal,
+        window_size_left,
+        window_size_right,
+        return_softmax,
+        gen_):
+   
+    print("flash_attn_triton_amd.py::varlen_fwd")
```

Comment:
Were these for debug, or did you want them to print out when a user calls this fn?

Reply:
I will clean it up.